src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs

```diff
@@ -193,7 +193,7 @@ private ModelProducer()
                         true,
                         out storeGeneratedPattern);
 
-                    if (storeGeneratedPattern != null)
+                    if (storeGeneratedPattern != null && (string)storeGeneratedPattern.Value == "Computed")
```

Comment:
Is this for #116?  @lewischeng-ms @challenh Can we have a sync on this?  How to represent `StoreGeneratedPattern.Identity` with OData annotation?